docs/en/deploy/Running-Alluxio-On-a-Cluster.md

```diff
@@ -31,7 +33,7 @@ The following sections describe how to install and configure Alluxio with a sing
 
 To deploy Alluxio in a cluster, first [download](https://alluxio.io/download) the Alluxio tar file,
 and copy it to every node (master node, worker nodes). Extract the tarball to the same path on
-every node.
+every node. You can download Alluxio tar file alternatively from [FTP](https://downloads.alluxio.io).
```

Comment:
@apc999 @haoyuan i think this is an important missing piece of the current user experience of going through our docs. most of docs will say "go download the allluxio binary" and users will navigate to the download page: https://www.alluxio.io/download/. from this page, the user can download a tarball to their local computer, but this is annoying for people launching a cluster in the cloud because users will need to take 2 steps to get the alluxio binary to their cluster: 1. download from website to local, 2. scp from local to each node. instead, i think it would be very useful to promote the 1 step flow of instructing each node to wget from the webserver at downloads.alluxio.io

Reply:
agree. I will actually have another pass on this doc. I will address your comments there